src/stories/Introduction.stories.mdx

```diff
@@ -0,0 +1,194 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```

Comment:
We can remove this introduction stories

Reply:
And also I think we can remove all next assets ('./assets/code-brackets.svg') which we are not going to use.